### ISSUE TITLE
[Feature Request] Add chat log recording in json format

### ISSUE THREAD

**Issue by kappaflow** [wmw9/rekoda#5]
Would be very useful to have the chat recorded in a [way it can be used](https://pastebin.com/raw/YDgRe6X4) with https://github.com/lay295/TwitchDownloader Chat Render functionality.

**Comment by evan-sm** [comment]
Wow, great tool! I am currently working on a Twitch chatbot, so I can implement this functionality later

**Comment by kappaflow** [comment]
[Twitch Leacher](https://github.com/Franiac/TwitchLeecher) got discounted, so [TwitchDownloader](https://github.com/lay295/TwitchDownloader) is probably the main tool for twitch vods downloading now.

**Comment by kappaflow** [comment]
Hi, do you still consider to work more on the app?
I actually use it, and it works mostly without any issues. There are some when the stream get turns on and off multiple times in a short period of time, in this case the file may be rewritten. Also, there is a possible issue when Internet stops working, it tries to reconnect only a short period of time.

**Comment by evan-sm** [comment]
Yes, I do plan! I'm really impressed someone is really using it 🤩 Thank you! Currently working on a small pet project and then I get back to `rekoda`

**Comment by kappaflow** [comment]
Thank you!
I am currently working on a real-time speech translation extension. If you are curious, you can check it out here:
https://chrome.google.com/webstore/detail/speech-translator/jodfjmaiakpnmeddgpeflpafebmlhppn
